dev/bin/kafka-setup.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+########
+# Full setup of test Managed Kafka instance and service accounts.
+# It is idempotent and can be run multiple times.
+#
+# Env vars:
+# - MANAGED_KAFKA_INSTANCE_NAME: set the managed kafka instance name (required)
+########
+
+. "$( dirname "$0" )/configure.sh" kafka
+
+# rhoas login
+function rhoas_login {
+  rhoas_logged_in=$( rhoas kafka describe -o json &> /dev/null && echo -n "yes" || echo -n "no" )
```

Comment:
Doesn't work in case kafka in current context doesn't exist.
What about using `rhoas status`?